spring-cloud-stream/src/main/java/org/springframework/cloud/stream/aggregate/AggregateApplicationBuilder.java

```diff
@@ -455,7 +462,7 @@ public ConfigurableApplicationContext run() {
 
 	}
 
-	@EnableAutoConfiguration
+	@ImportAutoConfiguration({ChannelBindingAutoConfiguration.class, EndpointAutoConfiguration.class})
```

Comment:
I think this is making things only marginally better. I remember that my original nitpick was above where this configuration if force fed. I don't like `@ImportAutoConfiguration` any better than `@EnableAutoConfiguration` on a framework level as framework should never force any auto-config. 
```
	private void addParentSources(Object[] sources) {
		if (!this.parentSources.contains(ParentConfiguration.class)) {
			this.parentSources.add(ParentConfiguration.class);
		}
		this.parentSources.addAll(Arrays.asList(sources));
	}
```
Maybe we could add some hooks to replace this config if user so want/need.